docker-compose-prod.yml

```diff
@@ -0,0 +1,63 @@
+# Version 3, most current, and recommended compose file version
+# https://docs.docker.com/compose/compose-file/compose-versioning/
+version: '3'
+
+services:
+  # First service
+  telescope:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
+    environment:
+      # a place-holder for the variable passed by CLI with a default value
+      # the passed or default value is a command to run `telescope`
+      - script=${script:-start}
+      # override the .env variable for `REDIS_URL`
+      - REDIS_URL=redis://redis:${REDIS_PORT}
+    depends_on:
+      - redis
+    ports:
+      - '$PORT:$PORT'
+    restart: always
+
+  # Second service
+  redis:
+    image: redis:latest
+    ports:
+      - '$REDIS_PORT:$REDIS_PORT'
+
+  # Third service
+  # SSO Identity Provider test service, https://simplesamlphp.org
+  # Access to the authentication page via http://localhost:8080/simplesaml or https://localhost:8443/simplesaml
+  ssoipd:
+    environment:
+      - SIMPLESAMLPHP_SP_ENTITY_ID=${SAML2_CLIENT_ID:-saml-poc}
+      - SIMPLESAMLPHP_SP_ASSERTION_CONSUMER_SERVICE=${SAML2_REDIRECT_URI}
```

Comment:
Note: these are changing in #640, so one of us will need to rebase and adjust (probably me, if you beat me into master, so remind me).